SPECS/python3/python3.spec

```
@@ -3,32 +3,25 @@
 # Automating the extraction of these alternate version strings has proven to be tricky,
 # with regards to tooling available in the toolchain build environment.
 # These will be manually maintained for the time being.
-%global majmin 3.9
-%global majmin_nodots 39
-# See Lib/ensurepip/__init__.py in Source0 for these version numbers
-%global pip_version 22.0.4
-%global setuptools_version 58.1.0
+%global majmin 3.12
```

Just a suggestion since we're making changes here to make it easier later:
```suggestion
%define majmin %(echo %{version} | cut -d. -f1-2)
```

---

thanks, will fix